src/core/remote.js

```diff
@@ -1610,7 +1610,7 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
   const request = new Request(this, 'book_offers');
 
   request.message.taker_gets = {
-    currency: Currency.json_rewrite(gets.currency, {force_hex: true})
```

Comment:
rippled can only handle 3 letter codes, or 40 hex characters for the currency string:
https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/handlers/BookOffers.cpp#L79
https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/UintTypes.cpp#L83-L99